en/orm/table-objects.rst

```diff
@@ -217,6 +228,17 @@ afterDelete
 
 The ``Model.beforeDelete`` event is fired after an entity has been deleted.
 
+afterDeleteCommit
+-----------------
+
+.. php:method:: afterDeleteCommit(Event $event, Entity $entity, ArrayObject $options)
+
+The ``Model.afterDeleteCommit`` event is fired after the transaction in which
```

Comment:
Same two issues as above:
``delete()`` and "... __a__ delete operation..." or "... __the__ delete operation..."